From 8c29afe59c395d1444518de2682151595e79c741 Mon Sep 17 00:00:00 2001
From: Roman Li <Roman.Li@amd.com>
Date: Thu, 13 Oct 2016 17:00:49 -0400
Subject: [PATCH 1308/1722] drm/amd/dal: fix underflow on Hawaii@144hz

Hardcoded min_mem_set_clock for DCE-8 to force powerplay
to increase bandwidth

Signed-off-by: Roman Li <Roman.Li@amd.com>

Change-Id: I9366bd1d3cfdc8cc8e5eb1c8e55c80de45d6106b
Acked-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/dce80/dce80_resource.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce80/dce80_resource.c b/drivers/gpu/drm/amd/dal/dc/dce80/dce80_resource.c
index fb6c039..8404ac6 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce80/dce80_resource.c
+++ b/drivers/gpu/drm/amd/dal/dc/dce80/dce80_resource.c
@@ -604,6 +604,7 @@ enum dc_status dce80_validate_bandwidth(
 {
 	/* TODO implement when needed but for now hardcode max value*/
 	context->bw_results.dispclk_khz = 681000;
+	context->bw_results.required_yclk = 250000 * MEMORY_TYPE_MULTIPLIER;
 
 	return DC_OK;
 }
-- 
2.7.4

